DeliBuddy/data/src/main/java/yapp/android1/data/entity/AddressMapper.kt

```diff
@@ -0,0 +1,29 @@
+package yapp.android1.data.entity
+
+import yapp.android1.domain.entity.Address
+
+object AddressMapper {
```

Comment:
```suggestion
object AddressDocumentResponseMapper {
```

훔..... 훔.... 어때요....?

Reply:
조금 헷갈릴 수도 있다고 느끼는 부분이 `AddressDocument`와 `KeywordDocument`를 둘 다 Address로 변환하는 부분이라서 바꾼다면 `AddressResponseMapper` 정도가 좋을 것 같은데 어떠신가요..?

Reply:
너무 좋습니다! 편한방향으로 갑쉬다